Project.toml

```diff
@@ -1,7 +1,7 @@
 name = "SNaQ"
 uuid = "c2bf7a07-44d2-4d11-b5e2-76dc5aa199d6"
 authors = ["Claudia Solis-Lemus <crsl4@users.noreply.github.com>", "Paul Bastide <pbastide@users.noreply.github.com>", "Cecile Ane <cecileane@users.noreply.github.com>", "and contributors"]
-version = "1.0.0"
+version = "1.1.0"
```

Comment:
If we're calling it SNaQ 2, would version 2.0.0 make sense? Though, with [semantic versioning guidelines](https://semver.org/), 1.1.0 seems more appropriate from my understanding of the things added